src/HttpOverStream.Client/DialMessageHandler.cs

```diff
@@ -62,6 +62,8 @@ protected override Task<Stream> CreateContentReadStreamAsync()
         protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
         {
             ValidateAndNormalizeRequest(request);
+
+            // Using (var stream?

```

Comment:
Should this comment stay?

Reply:
I left it in to prompt discussion on this PR. Should we have a `using` here? Should we have a `try { .. } catch { stream.dispose() }`?

Reply:
Actually i've noticed this is already on the list of things to fix from our last meeting:

* Add try/catch to the client dialmessagehandler and dispose the stream in the catch to make sure we dont leak stream if any exceptions occur

So i guess i can delete the comment